installer/roles/image_build/tasks/main.yml

```diff
@@ -42,7 +42,7 @@
   delegate_to: localhost
 
 - name: Build sdist builder image
-  docker_image:
+  community.general.docker_image:
```

Comment:
This wouldn't be backwards compatible with running on a bare 2.9.

Reply:
Understood. Will switch back & add note to the INSTALL.md that the requirement is only for ansible >= 2.10

Reply:
Switched back & marked conversation as resolved.